src/bundle/Resources/views/themes/admin/ui/component/input_text.html.twig

```diff
@@ -20,48 +20,60 @@
-                        <use xlink:href="{{ ibexa_icon_path('view') }}"></use>
+                    <svg class="ibexa-icon ibexa-icon--tiny-small">
+                        <use xlink:href="{{ ibexa_icon_path('discard') }}"></use>
                     </svg>
-                    <svg class="ibexa-icon ibexa-icon--small ibexa-input-text-wrapper__password-hide d-none">
-                        <use xlink:href="{{ ibexa_icon_path('view-hide') }}"></use>
-                    </svg>
-                </button>
-            {% elseif has_search %}
-                <button type="{{ search_button_type|default('submit') }}" class="btn ibexa-btn ibexa-btn--ghost ibexa-btn--no-text ibexa-input-text-wrapper__action-btn ibexa-input-text-wrapper__action-btn--search" tabindex="-1">
-                    <svg class="ibexa-icon ibexa-icon--small">
-                        <use xlink:href="{{ ibexa_icon_path('search') }}"></use>
-                    </svg>
-                </button>
-            {% elseif extra_btn.label is not empty %}
-                <button {{html.attributes(extra_btn.attr)}}>
-                    {{- extra_btn.label -}}
                 </button>
-            {% endif %}
-        {% endblock %}
+                {% if is_password_input %}
+                    <button
+                        type="button"
+                        class="btn ibexa-btn ibexa-btn--ghost ibexa-btn--no-text ibexa-input-text-wrapper__action-btn ibexa-input-text-wrapper__action-btn--password-toggler"
+                        tabindex="5"
+                    >
+                        <svg class="ibexa-icon ibexa-icon--small ibexa-input-text-wrapper__password-show">
+                            <use xlink:href="{{ ibexa_icon_path('view') }}"></use>
+                        </svg>
+                        <svg class="ibexa-icon ibexa-icon--small ibexa-input-text-wrapper__password-hide d-none">
+                            <use xlink:href="{{ ibexa_icon_path('view-hide') }}"></use>
+                        </svg>
+                    </button>
+                {% elseif has_search %}
+                    <button type="{{ search_button_type|default('submit') }}" class="btn ibexa-btn ibexa-btn--ghost ibexa-btn--no-text ibexa-input-text-wrapper__action-btn ibexa-input-text-wrapper__action-btn--search" tabindex="-1">
+                        <svg class="ibexa-icon ibexa-icon--small">
+                            <use xlink:href="{{ ibexa_icon_path('search') }}"></use>
+                        </svg>
+                    </button>
+                {% elseif extra_btn.label is not empty %}
+                    <button {{html.attributes(extra_btn.attr)}}>
+                        {{- extra_btn.label -}}
+                    </button>
+                {% endif %}
+            {% endblock %}
+        </div>
     </div>
+    {% block right_side_label %}
+        {% if right_side_label is defined %}
```

Comment:
I'm no expert in twig, but isn't here some potential problems in using same name for both variable and block name?